openupgrade_scripts/scripts/website_sale/14.0.1.0/post-migration.py

```diff
@@ -1,10 +1,38 @@
 # Copyright (C) 2021 Open Source Integrators <https://www.opensourceintegrators.com/>
 # Copyright 2021 ForgeFlow S.L.  <https://www.forgeflow.com>
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+import re
 
 from openupgradelib import openupgrade
 
 
+def extract_custom_product_page_term_conditions(env):
+    """Replace Terms and Conditions content in the new v14 template so as not to lose
+    content from previous versions if it has been customised."""
+    product_custom_text_view = env.ref("website_sale.product_custom_text")
+    if not product_custom_text_view:
+        return
+    product_custom_text_arch = product_custom_text_view.arch_db
+    product_custom_text_pattern = r'<p class="text-muted(.*?)<\/p>'
+    product_custom_text_matches = re.findall(
```

Comment:
So you don't have to access by index (which isn't a very clean practice. Also rename the variable to something like `product_custom_text_content` that explains better what the code is doing
```suggestion
    product_custom_text_matches, *_ = re.findall(
```

Reply:
Right! much clearer, thank you!